static/app/views/profiling/profilesProvider.tsx

```diff
@@ -86,7 +86,8 @@ function ProfileProviderWrapper(props: FlamegraphViewProps): React.ReactElement
   });
 
   const profileTransaction = useSentryEvent<EventTransaction>(
-    params.orgId,
+    // params.orgId does not seem to be set on dev, so provide a fallback
+    params.orgId || organization.slug,
```

Comment:
Can we just always use `organization.slug`?

Reply:
I dont see a reason why we couldnt